Makefile

```diff
@@ -1,9 +1,6 @@
 SHELL=bash
 
-ALL_EXCLUDE = third_party .git env build
-FORMAT_EXCLUDE = $(foreach x,$(ALL_EXCLUDE),-and -not -path './$(x)/*')
-
-PYTHON_SRCS=$(shell find . -name "*py" $(FORMAT_EXCLUDE))
+FASM2BELS_PYTHON_SRCS=$(shell find fasm2bels -name "*py")
```

Comment:
Why does this change?

Reply:
As all the python sources to reformat are in the fasm2bels directory, it was better to just find them in there instead of excluding all the others from the root dir IMO.

Reply:
`setup.py` also needs to be checked.

Reply:
that is checked in the make format command and is added to the list: https://github.com/SymbiFlow/symbiflow-xc-fasm2bels/blob/e5bde2f7e180e56c8872ac24fcfe3d5598d0f3b0/Makefile#L12